pkg/discovery/discovery.go

```diff
@@ -39,6 +40,24 @@ func NamespacedGVRs(ctx context.Context, cli discovery.DiscoveryInterface) ([]sc
 	return apiToGroupVersion(arls)
 }
 
+func NamespacedGVRsIgnoreAPIGroupErr(ctx context.Context, cli discovery.DiscoveryInterface, exclude filter.ResourceTypeMatcher) ([]schema.GroupVersionResource, error) {
```

Comment:
@tdmanv filter package is already importing discovery package, so wont be able to import filter package here to avoid import cycle.